### PR TITLE
adding current action to cli, to display current assignments for users

### PR DESCRIPTION
**Pull request by kevinleugers**
API endpoint is in place already for `/user/assignments/current`. Addressing [exercism.io issue #205](https://github.com/kytrinyx/exercism.io/issues/205) to add the ability to see current assignments from the cli.

### CODE REVIEW

**Comment by kytrinyx** [comment]
The build is failing on this one. Is there a missing VCR cassette?

**Comment by kevinleugers** [comment]
Yes, there is actually. I develop on windows using a Linux VM, so my localhost is actually the IP address of my VM session. 

I was afraid the cassette would cause problems with the IP address in there so I just rememoved it thinking it would be generated again. 

I hope that makes sense. I suppose I could just edit the cassette file after the fact so it points to localhost?

**Comment by kytrinyx** [comment]
Ah, got it. The CI server doesn't regenerate it -- I'd have to do that locally. I don't think it matters where it points, actually, though it's probably a good idea to edit it to point at whatever the other cassettes point to for consistency.

**Comment by kevinleugers** [comment]
Should be good to merge now :)

**Comment by kytrinyx** [comment]
Would you mind merging master into the branch?

File: lib/cli.rb [hunk 1]
```diff
@@ -159,7 +173,12 @@ def report(assignments)
         end
       end
     end
-  end
 
-  
+    def spacing(track)
```
Comment (mikegee) [inline lib/cli.rb:184]: Check out [String#ljust](http://www.ruby-doc.org/core-2.0.0/String.html#method-i-ljust)

File: lib/exercism/api.rb [hunk 2]
```diff
@@ -70,6 +70,14 @@ def list_stash(action)
       JSON.parse(response.body)
     end
 
+    def current
+      response = conn.get do |req|
+        req.url endpoint('user/assignments/current')
+        req.params = {:key => user.key}
+      end
+      response
```
Comment (markijbema) [inline lib/exercism/api.rb]: This line is redundant, and the whole variable can be left out.

**Comment by markijbema** [comment]
Cool, nice functionality :)

**Comment by kevinleugers** [comment]
@mikegee thanks! That's exactly what I needed there. @markijbema good catch, thank you.

**Comment by kytrinyx** [comment]
Sorry -- I keep missing this.

Would you mind merging in master again, and leaving a note in the discussion thread so that I am certain to see it?

**Comment by kevinleugers** [comment]
Sure! No problem. Will reply back when finished.

**Comment by kevinleugers** [comment]
@kytrinyx I believe this is ready now. Thanks!

**Comment by kytrinyx** [comment]
Thank you!

**Pull request merged**